### DISCUSSION TITLE
When is a player given XP in the leveling system?

### DISCUSSION THREAD

**Discussion by Dyljyn** [BlueDevelopersInc/DiscordSRVUtils#144] [Q&A] [answered]
Is it correct that XP is only given when a player sends a message, both in Discord and Minecraft?

I thought it would also be increasing while the user is online on the Minecraft server.

**Comment by BlueTree242** [comment, marked as answer]
No, it's message based

**Reply by Dyljyn** [reply]
How do you feel about making it online activity based as well?

**Reply by BlueTree242** [reply]
This is a discord plugin and you can't be active on discord without chatting

**Reply by Dyljyn** [reply]
Alright, totally fair :)
